src/expressions/antlr-interface/Expr.g4

```diff
@@ -28,6 +28,8 @@ expr
     | 'sum' '(' expr ')'                       # allTimeSum
     | 'sum_connections' '(' portFieldExpr ')'           # portFieldSum
     | 'sum' '(' from=shift '..' to=shift ',' expr ')'  # timeSum
+    | 'dual' '(' IDENTIFIER ')'                # dual
+    | 'reduced_cost' '(' IDENTIFIER ')'        # reducedCost
```

Comment:
I would say
```suggestion
    | FUNCTION '(' IDENTIFIER ')'           # function
```
Where FUNCTION is one of `dual`, `reduced_cost`, etc. The generated code will have only one ANTLR visitor, as opposed to N (=2) here.